src/core_modules/capture-core/reducers/descriptions/currentSelections.reducerDescriptionGetter.js

```diff
@@ -233,4 +237,12 @@ export const getCurrentSelectionsReducerDesc = (appUpdaters: Updaters) => create
         categoriesMeta: undefined,
         trackedEntityTypeId,
     }),
-}, 'currentSelections');
+}, 'currentSelections', {
+    complete: false,
+    categoryCheckInProgress: false,
+    orgUnitId: null,
+    programId: null,
+    trackedEntityTypeId: null,
+    categories: null,
+    categoriesMeta: null,
```

Comment:
I would be careful assigning null here, because this will override default function param assignments. Especially for categories and categoriesMeta which are saved as objects. I suggest assigning to undefined or not at all.

Reply:
very good point! thanks